### DISCUSSION TITLE
One image is like 1000 words

### DISCUSSION THREAD

**Discussion by spapas** [tfwright/live_admin#91] [Ideas]
Please add screenshots of this project on the readme.

thank you

**Comment by tfwright** [comment]
Hi @spapas and thanks for the feedback. I have indeed gotten this request before and agree would be really improve the README to feature some screenshots. My main reason for delaying is that I am not altogether pleased with the current look and feel and would like to tighten it up a good amount before releasing 1.0. If you know anyone with strong UI design skills, please feel free to direct them here 🙂

**Reply by kidbombay** [reply]
Hey I'd be happy to help the UI for this. I've built and designed many admin UIs.

**Reply by tfwright** [reply]
Hi @kidbombay thanks for volunteering. So far you're the only one so feel free! The only caveat is that I would like to avoid major changes to the UX/layout for now, so it would be pretty much CSS only (although of course minor changes would be fine). Once we have a "look" I would also be open to suggestions for more significant changes to UX. If you want to start by opening an issue with some screenshots of your proposed design we can start there?

**Reply by kidbombay** [reply]
Are you using Tailwind?

**Comment by spapas** [comment]
Hey @tfwright I think that even if they are not very beautiful some screenshots would give people a better idea of the capabilities and functionality of the project. A lot of people can quickly decide if this project would fit their needs by taking a peek at the screenshots without the need to installin it first!
